src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -759,6 +759,105 @@ void testPatchAuthDomain() throws Exception {
           .patchAuthDomainV2(
               IamResourceType.DATASNAPSHOT.getSamResourceName(), snapshotId.toString(), authDomain);
     }
+
+    @Test
+    void setResourceParent() throws ApiException, InterruptedException {
```

Comment:
To make it easier to maintain tests, I use `throws Exception`. Using specific exception types doesn't affect the test and ends up being a maintenance burden if the types ever change.
```suggestion
    void setResourceParent() throws Exception {
```